cmd/heimdall/main.go

```diff
@@ -28,6 +28,9 @@ func main() {
 
 func run() error {
 	config, err := loadConfig()
+	if err != nil {
```

Comment:
Nice catch. I quickly rushed out that last change and didn't even notice I missed this.

Reply:
Yellow squiggly lines to the rescue!